packages/http/src/http.ts

```diff
@@ -19,6 +19,7 @@ import { createWorkflow, WorkflowEvent } from '@deepkit/workflow';
 import type { ElementStruct, render } from '@deepkit/template';
 import { FrameCategory, Stopwatch } from '@deepkit/stopwatch';
 import { getSerializeFunction, hasTypeInformation, ReflectionKind, resolveReceiveType, SerializationError, serialize, serializer, ValidationError } from '@deepkit/type';
+import stream from 'stream';
```

Comment:
could we decouple this from nodejs? So that we also support Deno/etc

Reply:
@TheNightmareX exactly. I've already created an issue for that https://github.com/deepkit/deepkit-framework/issues/236